docs/manual/glossary.md

```diff
@@ -0,0 +1,78 @@
+---
+uid: glossary
+---
+
+# Glossary
+
+## D
+
+### Data channel
+
+A data channel is a "pipe" to send and receive random blobs of bytes.
+
+## L
+
+### Local media track
+
+A local media track is a [_media track_](#media-track) whose source is local to the peer connection, that is which has a local frame-producing [_media track source_](#media-track-source). In other words the track is a sender track, which produce frames to be sent to the remote peer.
+
+> See also
+>
+> - [_media track_](#media-track)
+> - [_remote media track_](#remote-media-track)
+
+## M
+
+### Media
+
+The term _media_ refers to either or both of _audio_ and _video_. It does **not** encapsulate [_data channels_](#data-channel).
```

Comment:
```suggestion
The term _media_ refers to either or both of _audio_ and _video_. It does **not** include [_data channels_](#data-channel).
```
Note sure "encapsulate" is correct here.

Reply:
Fixed.